garden/ros2_integration.md

```diff
@@ -74,9 +74,7 @@ ros2 topic echo /keyboard/keypress
 This command listens to the messages sent over the `/keyboard/keypress` topic.
 
 On the Gazebo window, press on the keyboard keys and you should
-find data on the listener terminal. This is what the terminals should look like:
-
-![exchange_messages](tutorials/ros2_integration/ros_ign.png)
```

Comment:
I don't know if this line and the image file should be deleted. The `ign` command in it is outdated, but seeing the ROS 2 topic list is still helpful. Unless we are replacing it with an updated image, I think having an image there is still more helpful than not having an image at all.

Reply:
So, when I looked at the image it seemed more confusing to me than beneficial with the differing namespace.

Reply:
This doesn't preclude one being added in future that is updated, but the goal is to avoid confusion and for the docs to be as accurate as possible.